mobile/src/main/java/com/gelakinetic/mtgfam/fragments/DecklistFragment.java

```diff
@@ -469,6 +470,51 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.deck_menu_legality: {
                 showDialog(DecklistDialogFragment.DIALOG_GET_LEGALITY, null, false);
             }
+            case R.id.deck_menu_export_dec: {
+                StringBuilder deck = new StringBuilder(30);
+                deck.append("// Generated by MTGFamiliar\n");
+                for (CompressedDecklistInfo info : mCompressedDecklist) {
+                    if (!"".equals(info.mName)) { /* Exclude headers from the decklist */
+                        if (info.mIsSideboard) { /* Exporting sideboards is a little different */
+                            deck.append("SB: ")
+                                    .append(info.getTotalNumber())
+                                    .append(' ')
+                                    .append(info.mName)
+                                    .append('\n');
+                            continue;
+                        }
+                        deck.append(info.getTotalNumber())
+                                .append(' ')
+                                .append(info.mName)
+                                .append('\n');
+                    }
+                }
+                String state = Environment.getExternalStorageState();
+                if (Environment.MEDIA_MOUNTED.equals(state)) {
+                    String fileName = mCurrentDeck;
+                    if ("".equals(fileName)) {
+                        fileName = "unnamed_deck";
+                    }
+                    File file = new File(Environment.getExternalStorageDirectory(), "MTGFamiliar" + File.pathSeparator + fileName + ".dec");
+                    if (!file.mkdirs()) {
+                        Toast.makeText(
```

Comment:
Please use the ToastWrapper class instead

Reply:
Fixed.